composer.json

```diff
@@ -2,10 +2,10 @@
     "require-dev": {
       "woocommerce/woocommerce-sniffs": "^0.1.3",
       "phpunit/phpunit": "^9.6.8",
-      "woocommerce/qit-cli": "^0.3.4",
       "squizlabs/php_codesniffer": "^3.7.2",
       "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-      "wp-coding-standards/wpcs": "^2.3"
+      "wp-coding-standards/wpcs": "^2.3",
+        "woocommerce/qit-cli": "^0.4.0"
```

Comment:
Just out of curiosity, may I know the reason why the `woocommerce/qit-cli` is moved to the bottom?

Reply:
I am not sure, I just used the composer command to update it from the terminal.